WelsonJS.Toolkit/WelsonJS.Launcher/Program.cs

```diff
@@ -188,16 +188,20 @@ public static void OpenWebBrowser(string url)
             string userDataDir = Path.Combine(GetAppDataPath(), "EdgeUserProfile");
             string remoteAllowOrigins = $"{resourceServerUri.Scheme}://{resourceServerUri.Host}:{resourceServerUri.Port}";
             int remoteDebuggingPort = devToolsUri.Port;
+            bool isAppMode = string.Equals(
+                GetAppConfig("ChromiumAppMode"),
+                "true",
+                StringComparison.OrdinalIgnoreCase);
             string[] arguments = {
-                $"\"{url}\"",
+                isAppMode ? $"\"--app={url}\"" : $"\"{url}\"",
                 $"--remote-debugging-port={remoteDebuggingPort}",
                 $"--remote-allow-origins={remoteAllowOrigins}",  // for security reason
                 $"--user-data-dir=\"{userDataDir}\""
             };
 
             Process.Start(new ProcessStartInfo
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Passing user-provided input directly to operating system command functions without validation introduces an OS command injection vulnerability. This allows attackers to inject and execute arbitrary commands on the host system. To remediate this, avoid constructing command lines using raw user input. Use the ProcessStartInfo class with ArgumentList to safely pass parameters without shell interpretation, and avoid calling cmd.exe /c when possible. Always validate or allowedlist input to ensure only safe values are used. Learn more: https://cwe.mitre.org/data/definitions/78.html